presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -60,7 +60,7 @@
     private boolean legacyOrderBy;
     private boolean legacyMapSubscript;
     private boolean optimizeMixedDistinctAggregations;
-    private boolean distributedSort = false;
+    private boolean distributedSort = true;
```

Comment:
Presumably we want to test both the default and with distributedSort=true, though